core/editable.js

```diff
@@ -1211,6 +1211,38 @@
 						return false;
 					}, this, null, 100 ); // Later is better – do not override existing listeners.
 				}
+
+				// Scroll to current ranges due to bug on iOS #498
+				if ( !this.isInline() && CKEDITOR.env.iOS && CKEDITOR.env.safari ) {
+					var touchListenerStatus, scrollListenerStatus;
+
+					this.on( 'textInput', scrollToRangesOnScroll, this, null, 10000 );
+					this.on( 'keydown', scrollToRangesOnScroll, this, null, 10000 );
+				}
+
+				function scrollToRangesOnScroll() {
+					var contentsWrapper = CKEDITOR.document.findOne( '#' + CKEDITOR.instances.editor.id + '_contents' );
```

Comment:
Can't we use `editor.container` here?

What's more, this code works **only** in instance name `editor`! The editor's instance is hardcoded.